zebrad/src/components/inbound.rs

```diff
@@ -46,6 +50,25 @@ mod tests;
 
 use downloads::Downloads as BlockDownloads;
 
+/// The number of bytes the [`Inbound`] service will queue in response to a single block or
+/// transaction request, before ignoring any additional block or transaction IDs in that request.
+///
+/// This is the same as `zcashd`'s default send buffer limit:
+/// <https://github.com/zcash/zcash/blob/829dd94f9d253bb705f9e194f13cb8ca8e545e1e/src/net.h#L84>
+/// as used in `ProcessGetData()`:
+/// <https://github.com/zcash/zcash/blob/829dd94f9d253bb705f9e194f13cb8ca8e545e1e/src/main.cpp#L6410-L6412>
+pub const GETDATA_SENT_BYTES_LIMIT: usize = 1_000_000;
```

Comment:
👍

It looked like the constant is 1000 in zcashd but it says elsewhere that that's in units of 1000 bytes.

Reply:
Yep, it's multiplied in the code before use.